### ISSUE TITLE
Do you have plans on the gim_EfficientLoFTR?

### ISSUE THREAD

**Issue by souxun2015** [xuelunshen/gim#10]
EfficientLoFTR is much faster than LoFTR.
https://zju3dv.github.io/efficientloftr/files/EfficientLoFTR.pdf

Do we have plans to retrain EfficientLoFTR with GIM?

**Comment by xuelunshen** [comment]
Of course, we are happy to see any matching network trained under gim to enhance performance. We plan to release the training code for gim later, as it takes quite some time for me alone to continually integrate new matching networks into gim training. Once the training code is released, everyone can improve their performance in gim on their own.

**Comment by javiabellan** [comment]
EfficientLoFTR with GIM would be awesome!

**Comment by xuelunshen** [comment]
@souxun2015 @javiabellan 
Sorry for the long wait, I have now released the code for video data generation and training network. Please refer to the [README](https://github.com/xuelunshen/gim?rgh-link-date=2024-12-21T14%3A44%3A53Z#%EF%B8%8F-video-preprocess) for relevant information.
